transport/server.cc

```diff
@@ -165,11 +165,12 @@ sstring to_string(const event::schema_change::change_type t) {
 
 sstring to_string(const event::schema_change::target_type t) {
     switch (t) {
-    case event::schema_change::target_type::KEYSPACE: return "KEYSPACE";
-    case event::schema_change::target_type::TABLE:    return "TABLE";
-    case event::schema_change::target_type::TYPE:     return "TYPE";
-    case event::schema_change::target_type::FUNCTION: return "FUNCTION";
-    case event::schema_change::target_type::AGGREGATE:return "AGGREGATE";
+    case event::schema_change::target_type::KEYSPACE:        return "KEYSPACE";
+    case event::schema_change::target_type::TABLET_KEYSPACE: return "KEYSPACE";
```

Comment:
Shouldn't "TABLET_KEYSPACE" be returned?
Also, please separate the cosmetic changes into a different patch. It makes review harder and I almost missed the above due to the noise.

Reply:
> Shouldn't "TABLET_KEYSPACE" be returned?

We're technically still modifying a KEYSPACE, and there is no separate identity like a tablet-keyspace in CQL, hence I left just a "KEYSPACE" string. As explained in the commit's title, this new enumerator `event::schema_change::target_type::TABLET_KEYSPACE` is only needed to "... differentiate processing path of a vnode vs tablets ks" - I rely on it later in the series.

> Also, please separate the cosmetic changes into a different patch. It makes review harder and I almost missed the above due to the noise.

Do you want me to only introduce `event::schema_change::target_type::TABLET_KEYSPACE` in this file in this commit and provide an additional commit that aligns all the `return "xxx";` statements?

Reply:
I reverted the formatting.